packages/ava/src/insight/insights/extractors/majority.ts

```diff
@@ -13,7 +13,7 @@ type MajorityParams = {
 
 const DEFAULT_PROPORTION_LIMIT = 0.6;
 
-export const findMajority = (values: number[], params?: MajorityParams): MajorityItem => {
+export const findMajority = (values: number[], params?: MajorityParams): MajorityItem | null => {
```

Comment:
Do u have to use the arrow function? There are some cases like this in these files

Reply:
done